Archive/Wiedijk100Theorems/CubingACube.lean

```diff
@@ -151,10 +150,10 @@ open Cube
 variable {ι : Type} {cs : ι → Cube (n + 1)} {i i' : ι}
 
 /-- A finite family of (at least 2) cubes partitioning the unit cube with different sizes -/
-@[protect_proj]
+-- @[protect_proj] -- Porting note: does not exist
```

Comment:
You need to add `protected` to each field of the structure instead. Alternatively, since this is just a leaf node, just drop it.

Reply:
I dropped it.